third_party/nvfuser/csrc/parallel_dimension_map.cpp

```diff
@@ -303,7 +303,7 @@ bool ParallelDimensionMap::equalDim(Val* dim1, Val* dim2) {
         dim2_def->as<UnaryOp>()->getUnaryOpType()))) {
     for (const auto i : c10::irange(dim1_def->inputs().size())) {
       (void)i; // Suppress unused variable warning
```

Comment:
Can't this line be removed now?

Reply:
I still see compilation errors with unused variables, so no I don't think so.

Reply:
It makes sense there was one before... but you're using `i` now... that's really strange, no?

Reply:
Aha!

Reply:
#2520